### PR TITLE
SAFRAN-952

### PR DESCRIPTION
**Pull request by orichalque**

### CODE REVIEW

File: addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/handlers/SetTargetSchemaHandler.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	}
+
+	/**
+	 * Sets the physical target {@link Annotation} of the {@link Reference}.
+	 * It is based on the {@link Namespace} of the {@link StructuredType} the {@link Reference} belongs to.
+	 * @param reference a {@link Reference}
+	 * @param physicalTarget a {@link String}
+	 */
+	private void setPhysicalTarget(Reference reference, String physicalTarget) {
+		TransactionalEditingDomain domain = TransactionUtil.getEditingDomain(reference);
```
Comment (orichalque) [inline addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/handlers/SetTargetSchemaHandler.java:64]: TransactionUtil.getEditingDomain()

Comment (orichalque) [reply addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/handlers/SetTargetSchemaHandler.java:64]: new EObjectQuery(reference).getSession().getTransactionalEditingDomain()

File: addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/propertyTesters/ReferencePropertyTester.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * {@link PropertyTester} for asserting that a {@link Reference} is bidirectionnal with two multiplicities.
+ * 
+ * @author Thibault Béziers la Fosse <a href="mailto:thibault.beziers-la-fosse@obeo.fr">thibault.beziers-la-fosse@obeo.fr</a>
+ *
+ */
+public class ReferencePropertyTester extends PropertyTester {
+
+	private static final String IS_NN = "isNN";
```
Comment (vrichard12) [inline addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/propertyTesters/ReferencePropertyTester.java:16]: why not calling it isMultiple or isMany as in ECore ?

File: commons/utils/plugins/org.obeonetwork.utils.sirius/src/org/obeonetwork/utils/sirius/propertyTesters/SiriusRepresentationPropertyTester.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		DRepresentation dRepresentation = SiriusUtil.findRepresentation(dRepresentationElement);
+		
+		if (dRepresentation != null ) {
+			DRepresentationQuery dRepresentationQuery = new DRepresentationQuery(dRepresentation);
+			
+			if (dRepresentationQuery.getRepresentationDescriptor() != null 
+					&& dRepresentationQuery.getRepresentationDescriptor().getDescription() != null
+					&& dRepresentationQuery.getRepresentationDescriptor().getDescription().getLabel() != null) {
+				
+				return dRepresentationQuery.getRepresentationDescriptor().getDescription().getLabel().equals(expectedValue);
```
Comment (vrichard12) [inline commons/utils/plugins/org.obeonetwork.utils.sirius/src/org/obeonetwork/utils/sirius/propertyTesters/SiriusRepresentationPropertyTester.java:48]: replace getLabel() by getName()

**Review by vrichard12 — CHANGES_REQUESTED** [review]

File: addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/propertyTesters/ReferencePropertyTester.java (hunk 2 above)
Comment (orichalque) [reply addons/structuredtypes/plugins/fr.gouv.mindef.safran.is.design/src/fr/gouv/mindef/safran/is/propertyTesters/ReferencePropertyTester.java:16]: Good call. I'll update it

**Review by vrichard12 — APPROVED** [review]

**Pull request merged**